vsts/pipelines/templates/_releaseJobTemplate.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: OryxLinux
   variables:
     skipComponentGovernanceDetection: true
-
+  timeoutInMinutes: 400
```

Comment:
really? :O

Reply:
lol, yeah, i am tired of these timeouts happening